### PR TITLE
feat: custom feed sorting

### PR DESCRIPTION
**Pull request by atymic**
As per https://github.com/spatie/laravel-feed/issues/71

- Updated config to add `sort_by` option, defaulted to `updated`
- Added `sortKey` property to the `Feed` class, as well as a `sortByDesc()` method that sorts by a given key
- Updated tests

### CODE REVIEW

File: src/Feed.php [hunk 1]
```diff
@@ -51,7 +57,7 @@ public function toResponse($request): Response
 
     protected function resolveItems($resolver): Collection
     {
-        $resolver = array_wrap($resolver);
+        $resolver = Arr::wrap($resolver);
```
Comment (atymic) [inline src/Feed.php]: `array_wrap()` is deprecated and will be remove in 5.9 so I swapped this out to the static version.

File: tests/__snapshots__/FeedTest__all_feed_items_have_expected_data__4.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<?php return '<?xml version="1.0" encoding="UTF-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
+                        <id>http://localhost/feedBaseUrl/feed4</id>
+                                <link href="http://localhost/feedBaseUrl/feed4"></link>
+                                <title><![CDATA[Feed 4]]></title>
+                                <updated>2016-01-05T00:00:00+01:00</updated>
+                        <entry>
+            <title><![CDATA[feedItemTitle]]></title>
+            <link rel="alternate" href="https://localhost/news/testItem1" />
+            <id>http://localhost/5</id>
```
Comment (atymic) [inline tests/__snapshots__/FeedTest__all_feed_items_have_expected_data__4.php]: Sorted by ID (desc)

**Comment by freekmurze** [comment]
I don't think this is necessary. Users can just return items in the order they want in the method specified in the `items` key in the config file.

**Comment by atymic** [comment]
No worries, I was just implementing the suggestion as per seb's comment:
https://github.com/spatie/laravel-feed/issues/71#issuecomment-395073509

I agree with your point :)

**Pull request closed without merging**